### DISCUSSION TITLE
Vehicle Status List?

### DISCUSSION THREAD

**Discussion by Hpnotiq1** [Courseplay/courseplay#7435] [Q&A]
First of all I want to apologize if this has ever been asked before, I just don't know what it is exactly called to be able to search for an answer. Anyways, is there a way to disable this what I call the vehicle status list?

![Screenshot_2](https://user-images.githubusercontent.com/57565309/129498765-3e70cbdf-d4f2-42a3-a9e3-b9d444f350ed.jpg)

**Comment by Tensuko** [comment]
No, not yet. But dude you got 8 finished vehicle... Why not care for them ?
Click the button in front of the message and stop them.

**Comment by Hpnotiq1** [comment]
It’s the only way to keep my belt system running without having to go to it all the time to engage them. 🤷‍♀️
